Rn entry 2022.09.0 (#3584)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

#### Clone authentication policies are GA in Preview

Creating an authentication policy from scratch is a manual, error-prone task because you need to visually copy existing rules into the new policy. Okta now offers the ability to clone a policy. You can use either the Admin Console or the new Clone a Policy operation on the Policy API. See [Clone a Policy](/docs/reference/api/policy/#clone-a-policy).<!--OKTA-515109-->

#### Improvements to the self-service password reset experience are GA in Preview

Previously, the self-service password reset (SSPR) flow created unnecessary friction in the user experience. The newly enhanced SSPR feature introduces a seamless magic link experience for password reset emails. Users no longer need to provide consent when using the same browser. After a successful password reset where the password meets the application’s assurance policy, the user is signed directly to the app. See [Configure the Email authenticator](https://help.okta.com/okta_help.htm?type=oie&id=csh-configure-email).<!--OKTA-499514-->

#### Password restriction for shared SWA app accounts is GA in Production

> For SWA apps with an account sign-in option set to Users share a single username and password set by administrator, only super admins or app admins with permissions for that app can view the password.<!--OKTA-513421-->

Reviewer: This one was hard to digest. I'm suggesting putting the setting in bold so that it's easier to read.
```suggestion
For SWA apps with an account sign-in option set to **Users share a single username and password set by administrator**, only super admins or app admins with permissions for that app can view the password.<!--OKTA-513421-->
```

#### Developer documentation updates in 2022.09.0

> * [developer.okta.com](developer.okta.com) has a new main navigation menu, which makes navigating around the content easier and more intuitive. Instead of showing the entire sitemap in the menu at all times, each major area is now accessed via a submenu, which shows only the relevant menu sections at one time. In addition, clicking on each menu node will show a contextual landing page (either a defined custom page, or an auto-generated one if none exists), and you can now toggle the menu between hidden and visible states if desired.

Reviewer: ```suggestion
* [developer.okta.com](developer.okta.com) has a new main navigation menu, which makes navigating around the content easier and more intuitive. Instead of showing the entire sitemap in the menu at all times, each major area is now accessed through a submenu that shows only the relevant menu sections at one time. In addition, clicking each menu node shows a contextual landing page (either a defined custom page or an auto-generated one if none exists), and you can now toggle the menu between hidden and visible states, if desired.
```
